#
#  We want sparc/i386 to match locations for their 32 bit support when building
#  multilib.  For now use the more generic /32.  This has not been filed
#  upstream.
#
--- gcc-5.3.0.orig/gcc/config/sparc/t-sol2	Wed Apr 30 09:10:44 2014
+++ gcc-5.3.0/gcc/config/sparc/t-sol2	Tue Jan  5 10:50:03 2016
@@ -1,4 +1,4 @@
 MULTILIB_OPTIONS = m32/m64
-MULTILIB_DIRNAMES = sparcv8plus sparcv9
+MULTILIB_DIRNAMES = 32 sparcv9
 MULTILIB_MATCHES =
 MULTILIB_OSDIRNAMES = . sparcv9
